### PR TITLE
Unlock Psalm level 6

### PR DESCRIPTION
**Pull request by leofeyer**
| Q                | A
| -----------------| ---
| Fixed issues     | n/a
| Docs PR or issue | n/a

### CODE REVIEW

File: core-bundle/src/Command/MigrateCommand.php [hunk 1]
```diff
@@ -278,6 +277,8 @@ static function ($hash) use ($commandsByHash) {
                 return false;
             }
         }
+
+        throw new \RuntimeException('This should never happen');
```
Comment (bytehead) [inline core-bundle/src/Command/MigrateCommand.php:281]: Won't PHPStan report here? 🙃

Comment (leofeyer) [reply core-bundle/src/Command/MigrateCommand.php:281]: It did not on my local system. 🤷‍♂

Comment (bytehead) [reply core-bundle/src/Command/MigrateCommand.php:281]: Funny. I remember it did while the developers meeting with unreachable statement 🤷‍♂

Comment (leofeyer) [reply core-bundle/src/Command/MigrateCommand.php:281]: That was a different file. 😄

Comment (bytehead) [reply core-bundle/src/Command/MigrateCommand.php:281]: But it seems to work also in the pipeline.

**Pull request closed without merging**